test/integration/racy-reconcile/00-install-instance.yaml

```diff
@@ -1,10 +0,0 @@
-apiVersion: kudo.dev/v1beta1
```

Comment:
This test checks that you can install `Instance` **before** the `OperatorVersion`. This doesn't work anymore